src/DI/Loader/DoctrineAnnotationLoader.php

```diff
@@ -224,28 +228,35 @@ protected function parseControllerMethodsAnnotations(Controller $controller, Ref
 
 				// Parse @RequestParameters ================
 				if ($annotation instanceof RequestParameters) {
-					foreach ($annotation->getParameters() as $p) {
-						$parameter = $schemaMethod->addParameter($p->getName(), $p->getType());
-						$parameter->setDescription($p->getDescription());
-						$parameter->setIn($p->getIn());
-						$parameter->setRequired($p->isRequired());
-						$parameter->setDeprecated($p->isDeprecated());
-						$parameter->setAllowEmpty($p->isAllowEmpty());
+					foreach ($annotation->getParameters() as $parameter) {
+						$this->addEndpointParameterToSchemaMethod($schemaMethod, $parameter);
 					}
 
 					continue;
 				}
 
-				// Parse @Response ================
+				// Parse #[RequestParameter] ================
+				if ($annotation instanceof RequestParameter) {
+					$this->addEndpointParameterToSchemaMethod($schemaMethod, $annotation);
+
```

Comment:
Can you please drop this empty line before `continue`?

Reply:
Sure :+1:

Reply:
Done :heavy_check_mark: